color_interpolate/analysis_options.yaml

```
@@ -1,4 +1,14 @@
 include: package:flutter_lints/flutter.yaml
 
 # Additional information about this file can be found at
-# https://dart.dev/guides/language/analysis-options
+# https://dart.dev/guideinclude: package:lints/recommended.yaml
+
+analyzer:
+  exclude: [build/**]
```

Would be pretty good to allow leaving TODOs along the code-base.

```suggestion
  errors:
    # Allow having TODOs in the code
    todo: ignore
  exclude: [build/**]
```